dbeam-core/src/test/java/com/spotify/dbeam/avro/JdbcAvroRecordTest.java

```diff
@@ -175,4 +174,48 @@ public void shouldEncodeResultSetToValidAvro()
     Assert.assertEquals(Coffee.COFFEE1, actual);
   }
 
+  @Test
+  public void checkOutputSchemaContainsInputSchemaDocs()
+      throws ClassNotFoundException, SQLException, IOException {
+    ResultSet rs = DbTestHelper.createConnection(CONNECTION_URL)
+        .createStatement().executeQuery("SELECT * FROM COFFEES");
+    String[] fieldNames = {"COF_NAME", "SUP_ID", "PRICE"};
+    String[] fieldDocs = {
+      "Input field COF_NAME doc", "Input field SUP_ID doc", "Input field PRICE doc"
+    };
+
+    final String recordName = "COFFEE";
+    final String recordDoc = "Input record doc";
+    final String recordNamespace = "Input record namespace";
+    Schema inputSchema = createRecordSchema(recordName, recordDoc, recordNamespace, fieldNames,
+        fieldDocs);
+
+    Schema schema =
+        JdbcAvroSchema.createAvroSchema(
```

Comment:
As mentioned earlier: we might not need call (and test) `JdbcAvroSchema.createAvroSchema` when `inputSchema` is present.